flocx_market/api/bid.py

```diff
@@ -0,0 +1,44 @@
+from flask_restful import Resource
+from flocx_market.db.sqlalchemy.bid_api import BidApi
+from flask import request
+
+
+class Bid(Resource):
+
+    @classmethod
+    def get(cls, marketplace_bid_id):
+        if marketplace_bid_id is None:
+            return BidList.get()['bids']
+        bid = BidApi.find_by_id(marketplace_bid_id)
+        if bid:
+            return bid.as_dict()
+        return {'message': 'Bid not found'}, 404
+
+    @classmethod
+    def post(cls):
+        data = request.get_json(force=True)
+        bid = BidApi(**data)
+        bid.save_to_db()
+        return bid.as_dict(), 201
+
+    @classmethod
+    def delete(cls, marketplace_bid_id):
+        bid = BidApi.find_by_id(marketplace_bid_id)
+        if bid:
+            bid.delete_from_db()
+            return {'message': 'Bid deleted.'}
+        return {'message': 'Bid not found.'}, 404
+
+    @classmethod
+    def put(cls, marketplace_bid_id):
```

Comment:
This should return a 404 error if the client attempts to `PUT` against an offer that doesn't exist.

Reply:
(I've fixed this for offers in #65)